src/Compilers/Core/Portable/PortableShim.cs

```diff
@@ -413,6 +413,11 @@ internal static class Encoding
                 .GetTypeInfo()
                 .GetDeclaredMethod(nameof(GetEncoding), paramTypes: new[] { typeof(int) })
                 .CreateDelegate<Func<int, System.Text.Encoding>>();
+
+            internal static readonly Func<System.Text.Encoding> Default = Type
```

Comment:
We can't make this change because `Encoding.Default` isn't a part of the portable API surface.